### PR TITLE
CLDR-17751 add ZWG, drop ZWL

### PR DESCRIPTION
**Pull request by srl295**
- update ZWG per ISO amendment
- retain USD as valid due to feedback and articles
- update English

CLDR-17751

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by AEApple** [comment]
Where is ZWG?

**Comment by macchiati** [comment]
My thought too!

**Review by macchiati — APPROVED** [review]

**Comment by srl295** [comment]
> My thought too!

yea… missing something important!

**Review by AEApple — APPROVED** [review]

**Comment by AEApple** [comment]
This actually looks wrong in staging? ZWL and ZWG don't appear properly split out. Also I expect ZWL to have a date range, but only one date is showing?

**Comment by macchiati** [comment]
What are you seeing on staging?

(if there is no "to" value, that just means that there is no set 'termination' date. That's typically for the current currency valid for a country)

File: common/main/en.xml [hunk 1]
```diff
@@ -6636,6 +6636,11 @@ annotations.
 				<displayName count="one">Zimbabwean dollar (1980–2008)</displayName>
 				<displayName count="other">Zimbabwean dollars (1980–2008)</displayName>
 			</currency>
+			<currency type="ZWG">
+				<displayName>Zimbabwean Gold</displayName>
+				<displayName count="one">Zimbabwean Gold</displayName>
+				<displayName count="other">Zimbabwean Gold</displayName>
```
Comment (roozbehp) [inline common/main/en.xml:6642]: I think the cases for one and other should use a lowercase "gold".

**Review by roozbehp — CHANGES_REQUESTED** [review]

**Review by roozbehp — APPROVED** [review]

**Review by roozbehp — APPROVED** [review]

**Comment by srl295** [comment]
@roozbehp appreciate your re-review. Github was down when I made the other, so i forgot that it was still in progress.

**Comment by srl295** [comment]
@macchiati got a null pointer in `en_Dsrt` - it was trying to compare it as a logical group

**Comment by srl295** [comment]
I found the issue. CountItems silently completed because I passed the -Dmethod parameter incorrectly. Noted this on https://unicode-org.atlassian.net/browse/CLDR-17555

**Review by roozbehp — APPROVED** [review]

**Pull request merged**